test/server/cards/02_SHD/units/DoctorPershingExperimentingWithLife.spec.ts

```diff
@@ -0,0 +1,86 @@
+                        spaceArena: ['tieln-fighter']
+                    },
+                    player2: {
+                        groundArena: ['frontier-atrt'],
+                        spaceArena: ['cartel-spacer']
+                    }
+                });
+            });
+
+            it('should deal 1 damage to any selected friendly unit and draw a card', function () {
+                const { context } = contextRef;
+
+                const reset = () => {
+                    context.player2.passAction();
+                    context.doctorPershing.exhausted = false;
+                };
+
+                // Trigger the ability
+                context.player1.clickCard(context.doctorPershing);
+                context.player1.clickPrompt('Draw a card');
+                expect(context.player1).toBeAbleToSelectExactly([context.doctorPershing, context.wampa, context.tielnFighter]);
+                expect(context.player1).not.toHaveChooseNoTargetButton();
+
+                // Target a space friendly unit
+                context.player1.clickCard(context.tielnFighter);
+                expect(context.doctorPershing.exhausted).toBeTrue();
+                expect(context.tielnFighter).toBeInLocation('discard');
+                expect(context.player1.hand.length).toBe(1);
+                expect(context.player2.hand.length).toBe(0);
+
+                // Trigger the ability again
+                reset();
+                context.player1.clickCard(context.doctorPershing);
+                context.player1.clickPrompt('Draw a card');
+                expect(context.player1).toBeAbleToSelectExactly([context.doctorPershing, context.wampa]);
+                expect(context.player1).not.toHaveChooseNoTargetButton();
+
+                // Target a ground friendly unit
+                context.player1.clickCard(context.wampa);
+                expect(context.doctorPershing.exhausted).toBeTrue();
+                expect(context.wampa.damage).toBe(1);
+                expect(context.player1.hand.length).toBe(2);
+                expect(context.player2.hand.length).toBe(0);
+
+                // Trigger the ability again
+                reset();
+                context.player1.clickCard(context.doctorPershing);
+                context.player1.clickPrompt('Draw a card');
+                expect(context.player1).toBeAbleToSelectExactly([context.doctorPershing, context.wampa]);
+                expect(context.player1).not.toHaveChooseNoTargetButton();
+
+                // Target self
+                context.player1.clickCard(context.doctorPershing);
+                expect(context.doctorPershing.exhausted).toBeTrue();
+                expect(context.doctorPershing.damage).toBe(1);
+                expect(context.player1.hand.length).toBe(3);
+                expect(context.player2.hand.length).toBe(0);
+            });
+
+            it('should not be available if Doctor Pershing is exhausted', function () {
```

Comment:
It looks like you've picked up this pattern already but just a note - we have this unfortunate issue where the Jasmine tests are really slow and heavy, and we have finite Github Actions time in the free tier. So it's bad test practice but we tend to consolidate tests into as few as reasonably possible. In this case I think this test could be easily appended to the above one?